tests/pyoozie/test_builder.py

```diff
@@ -153,18 +154,42 @@ def test_workflow_builder(tmpdir):
             xml=actual_xml,
         ))
 
-    # Does it throw an exception on a bad name?
+    # Does it throw an exception on a bad workflow name?
```

Comment:
Instead of commenting sections of the test, could you split these into smaller tests with descriptive names? e.g. `test_builder_raises_on_bad_workflow_name`